telemetry/telemetry.go

```diff
@@ -209,3 +210,19 @@ func getflag(name string) string {
 	})
 	return val
 }
+
+func validateSHA256(file string) {
```

Comment:
Seems this function is returning void, does main function need to break if validation is failed?

Reply:
Called in block that will exit in new PR: https://github.com/sonic-net/sonic-gnmi/pull/186